src/Chapter2.hs

```diff
@@ -336,7 +348,7 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList start end list = take (1 + end-start) (drop start list)
```

Comment:
I think, here you need to add a few checks on the correct input first, because you could get the wrong results in some cases 🙂